certified-connectors/CloudConvert/apiProperties.json

```diff
@@ -0,0 +1,39 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "564",
```

Comment:
I guess this needs to be set later in ISV Studio? Just removed it.

Reply:
Hi @josiasmontag! Thanks for all the fixes! It'll be used later, yes. Could you please change the value to an empty string instead of null? So that it'll be `"clientId": ""`.